packages/client/helpers/build.ts

```diff
@@ -37,6 +37,7 @@ const edgeRuntimeBuildConfig: BuildOptions = {
   outfile: 'runtime/edge',
   bundle: true,
   minify: true,
+  format: 'esm',
```

Comment:
This could be considered a breaking change, as it might have unintended side effects. I suggest that we would add `edge` to become a folder, with both `cjs` and `esm` files that have respective entrypoints in a `package.json`.